integr8ly/workload-web-app
# DEL-329 Check AMQ console

**What**
* Login to the AMQ console using the customer-admin01 user
* Verify AMQ console availability

**To verify**
1. Checkout this branch
1. Run the below command
   ```
   export WORKLOAD_WEB_APP_IMAGE=quay.io/<replace-with-your-repo>/workload-app
   export GRAFANA_DASHBOARD=true
   make local/build-deploy
   ```
1. Wait for the app to get deployed
1. Login to the grafana and verify if the AMQ Console panel is present in workload-app dashboard
